client/implementation/src/main/java/io/smallrye/graphql/client/typesafe/impl/reflection/ParameterInfo.java

```diff
@@ -45,6 +46,8 @@ private String graphQlInputTypeName(TypeInfo type) {
     }
 
     private String simpleInputTypeName(TypeInfo type) {
+        if (type.isAnnotated(Input.class))
+            return type.getAnnotation(Input.class).value();
         if (type.isAnnotated(Name.class))
             return type.getAnnotation(Name.class).value() + "Input";
```

Comment:
Only append 'Input' on classname. same as server

Reply:
Okay... I'm not really convinced that this is the best way to go, but consistency with the server beats any argument ;-)